### PR TITLE
9.0 hierarchical

### PR DESCRIPTION
**Pull request by thibaultrey**
Add manufacturing hierarchical module that link workcenter to a parent workcenter.

### CODE REVIEW

File: mrp_workcenter_hierarchical/README.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Description
+===========
+
+See __openerp__.py
```
Comment (pedrobaeza) [inline mrp_workcenter_hierarchical/README.rst]: This is not correct. The **openerp** should be put here and remove it from the manifest.

File: mrp_workcenter_hierarchical/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (pedrobaeza) [inline mrp_workcenter_hierarchical/__init__.py]: Use short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: mrp_workcenter_hierarchical/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+
+{
+    'name': 'MRP Workcenter Hierarchical',
+    'version': '0.5',
```
Comment (pedrobaeza) [inline mrp_workcenter_hierarchical/__openerp__.py]: 9.0.1.0.0

**Comment by pedrobaeza** [comment]
What is the functional case that this covers? It's only for information purposes?

File: mrp_workcenter_hierarchical/i18n/mrp_workcenter_group.pot [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Translation of OpenERP Server.
```
Comment (sebastienbeau) [inline mrp_workcenter_hierarchical/i18n/mrp_workcenter_group.pot]: rename translation file as module have been renamed

File: mrp_workcenter_hierarchical/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'summary': "Organise Workcenters by section",
+    'category': 'Manufacturing',
+    'depends': [
+        'mrp_operations',
+    ],
+    'description': """
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (sebastienbeau) [inline mrp_workcenter_hierarchical/__openerp__.py]: Move everything in the readme file

File: mrp_workcenter_hierarchical/static/description/index.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (sebastienbeau) [inline mrp_workcenter_hierarchical/static/description/index.html]: Remove that file as we use the readme one

**Comment by LoisRForgeFlow** [comment]
@thibaultrey @florian-dacosta branch has been deleted. Good to close?

**Comment by LoisRForgeFlow** [comment]
Closing as this is stale. Please reopen if you think still is relevant.

**Pull request closed without merging**